CHANGELOG.md

```diff
@@ -1,5 +1,43 @@
 # Changelog
 
+## 0.3
+Released on May 3, 2023.
+
+### Key Features
+* Introduces the `aqueduct.llm_op` API and the `aqueduct-llm` package. Aqueduct
+    now has support for invoking LLMs with a single API call and comes with
+    pre-built Docker images optimized for executing LLMs on Kubernetes. The
+    `llm_op` API supports both ad hoc execution, as pictured below, as well as
+    batch execution over a list of inputs or a Pandas `Series`. See our
+    documentation for more details. **TODO: link the word documentation**
+    ```python
+    import aqueduct as aq
+
+    llm_op = aq.llm_op('vicuna-7b', engine='k8s')
+    llm_op('What is the best LLM?')
```

Comment:
Let's change the variable name to sth else as it conflicts with the module name...
Let's name it `vicuna`